SU2_CFD/src/integration/CIntegration.cpp

```diff
@@ -172,6 +170,9 @@ void CIntegration::Space_Integration(CGeometry *geometry,
           solver_container[MainSolver]->BC_HeatFlux_Wall(geometry, solver_container, conv_bound_numerics, visc_bound_numerics, config, iMarker);
         }
         break;
+      case SMOLUCHOWSKI_MAXWELL:
```

Comment:
By the way I do not see the catalytic boundaries being called here, maybe they got lost while merging with develop?